chapter7.adoc

```diff
@@ -124,13 +124,16 @@ When `SDICN` is not the number of an implemented interrupt controller, or is
 zero then the following rules apply to all privilege modes:
 
 * The S-level external interrupt pending signal indicated in `mip.SEIP` is 0.
-* All `siselect` values that access IMSIC registers are reserved.
+* All non-custom values of `siselect` designate an _inaccessible_ register. When
```

Comment:
Is this correct when there are users of Sscsrind other than AIA?

Reply:
It should have been a bit more precise. "All non-custom values that access IMSIC registers"

Reply:
updated.